satellite/metainfo/metainfo.go

```diff
@@ -120,6 +120,31 @@ func (endpoint *Endpoint) SegmentInfo(ctx context.Context, req *pb.SegmentInfoRe
 	return &pb.SegmentInfoResponse{Pointer: pointer}, nil
 }
 
+// StorageNodeInfo returns address of storage nodeID
+func (endpoint *Endpoint) StorageNodeInfo(ctx context.Context, req *pb.StorageNodeInfoRequest) (resp *pb.StorageNodeInfoResponse, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	nodes, err := endpoint.cache.GetAll(ctx, req.NodeId)
```

Comment:
Who will be client of this method (we or users)? if users, you need to add authentication.